packages/x-components/src/views/home/Home.vue

```diff
@@ -154,7 +162,7 @@
               </template>
             </SortDropdown>
 
-            <RenderlessExtraParams #default="{ value, updateValue }" name="store">
+            <RenderlessExtraParams v-slot="{ value, updateValue }" name="store">
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
            <RenderlessExtraParams #default="{ value, updateValue }" name="store">
```

Reply:
good catch, thanks!